torchtext/experimental/models/xlmr_transform.py

```diff
@@ -0,0 +1,28 @@
+import os
+import torch.nn as nn
+from torchtext.experimental.transforms import sentencepiece_tokenizer
+from torchtext.experimental.vocab import load_vocab_from_file
+from typing import List
+
+
+class XLMRTransform(nn.Module):
+    """XLM-R encode transform."""
+
+    def __init__(self, tokenizer, vocab):
+        super(XLMRTransform, self).__init__()
+        self.tokenizer = tokenizer
+        self.vocab = vocab
+
+    @classmethod
+    def from_pretrained(cls, directory='./',
+                        tokenizer_file="sentencepiece.bpe.model",
+                        vocab_file='vocab.txt'):
+        filepath = os.path.join(directory, tokenizer_file)
+        tokenizer = sentencepiece_tokenizer(filepath)
+        filepath = os.path.join(directory, vocab_file)
+        with open(filepath, 'r') as f:
+            vocab = load_vocab_from_file(f)
```

Comment:
Will all `pretrained` XLMRTransform` instances that we provide contain a `sentencepiece_tokenizer` and use a vocab loaded via `load_vocab_from_file`? If not, I see two options:
- try to make this code generic on the tokenizer type (which can be annoying to handle and brittle)
- remove this method altogether from `XLMRTransform` and instead let if be a free-function elsewhere.